apps/package.json

```diff
@@ -238,6 +238,7 @@
     "qrcode.react": "^0.8.0",
     "query-string": "4.1.0",
     "react-csv": "^1.0.14",
+    "react-debounce-input": "^3.2.2",
```

Comment:
Why not just use lodash.debounce?

Reply:
https://lodash.com/docs/4.17.15#debounce

Reply:
Summarizing offline conversation- we're okay with adding this dependency because it solves the exact problem we have (whereas using lodash debounce would require more custom logic in this class), and we think it will be useful in other places as well.